### PR TITLE
Link image movement

### PR DESCRIPTION
**Pull request by hotblack86**

### CODE REVIEW

**Review by michaelnguyen974 — APPROVED** [review]

**Review by matharotheelf — APPROVED** [review]
Good work!

File: src/hero.py [hunk 1]
```diff
@@ -17,6 +16,16 @@ def move(self, dx=0, dy=0):
         if not self.collide_with_walls(dx, dy) and not self.collide_with_enemy(dx, dy):
             self.x += dx
             self.y += dy
+        #Changes link image on each arrow key push
```
Comment (CKKH) [inline src/hero.py:19]: Nice!

**Review by CKKH — APPROVED** [review]
Looks great!

File: src/hero.py (hunk 1 above)
Comment (michaelnguyen974) [reply src/hero.py:19]: Tasty

**Pull request merged**